core/settings/base.py

```diff
@@ -32,6 +32,7 @@
     "rest_framework_simplejwt.token_blacklist",
     "drf_yasg",
     "taggit",
+    "django_filters",
```

Comment:
Where are you using this?

Reply:
Used django_filters to implement filtering